File: Vol.13(2200-2299)/#2201 No Brainer/p2201.cpp
#include <iostream>
using namespace std;

int main() {
	int	N,	a,	b;
	cin >> N; 
	while( N-- ) {
		cin >> a >> b;
		cout << ( ( a >= b ) ? "MMM BRAINS" : "NO BRAINS" ) << endl; 
	} 
} 

File: 范例/Accpt/22xx/p2201.pas
PROGRAM p2201;

VAR
        a,b     :integer;
        n       :integer;
        i       :integer;

BEGIN
readln(n);
for I:=1 to n do
        begin
        readln(a,b);
        if a<b then writeln('NO BRAINS') else writeln('MMM BRAINS');
        end;
END.